doc_src/en/HowTo_UseTM.xml

```diff
@@ -165,6 +165,25 @@
 	  one of its subfolders and the translated data will be immediately be
 	  available for matching purposes.</para>
 	  </listitem>
+
+          <listitem>
+            <para>If you switch to paragraph segmentation in a project that originally used <link
+			    linkend="dialogs.project.properties.options.segmentation"
+			    endterm="dialogs.project.properties.options.segmentation.title"/>,
+		    you can have OmegaT present matches assembled from existing segments by enabling the
+		    <link
+	    linkend="dialog.preferences.tm.matches.paragraph.from.segmented.tmx"
+	    endterm="dialog.preferences.tm.matches.paragraph.from.segmented.tmx.title"/> option in addition to
+			showing regular fuzzy matches.</para>
+			<para> OmegaT will try to segment the source text into sentences,
+			apply fuzzy matching on the segmented memory entries, and then assemble a paragraph from those matches.
+		    No origin is shown for the match because the assembled paragraph may not exist in the
```

Comment:
#1226 proposes the change to show a list of files from which compose the paragraph. When merged the #1226, the description here become obsolete.

for example,

```suggestion
		    The origins of the matches can be multiple files or memories because the assembled paragraph may not exist in the
```